action/protocol/staking/bucket_pool_test.go

```diff
@@ -152,7 +148,53 @@ func TestBucketPool(t *testing.T) {
 			r.Equal(count, pool.Count())
 		}
 	}
+	c, err = ConstructBaseView(sm)
```

Comment:
var testGreenland bool
if !testGreenland && v.postGreenland {
    c, err = ConstructBaseView(sm)
    _, err = sm.PutState()
    testGreenland = true
}

then don't need to copy the test code below